### PR TITLE
docs: Update installation instructions for kyverno aws adapter

### PR DESCRIPTION
**Pull request by kbeniwal**
Fixes: NDEV-15397

To-dos:

- [x] Add Kyverno “official documentation” hyperlink.
- [x] Account ID to be replaced in the ARN, in my-policy.json.
- [x] Account ID to be replaced in the ARN, in “Creating the IAM Role” command.
- [x] Clear indication of the Account ID part of ARN that needs to be replaced in myvalues.yaml file.
- [x] Fix grammatical/style errors.
- [x] Changing namespace does not lead to failure. Or specific statements to avoid that error.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -64,51 +67,53 @@ EOF
 aws iam create-policy --policy-name kyverno-aws-adapter-policy --policy-document file://my-policy.json
```
Comment (kumar-mallikarjuna) [inline docs/getting_started.md:67]: Good to change `kyverno-aws-adapter-policy` to `<aws-policy-name>`. In our tenant, we have multiple such policies, each adapter user creates their own.

File: docs/getting_started.md [hunk 2]
```diff
@@ -64,51 +67,53 @@ EOF
...
 
 ### Creating the IAM Role
+
 Create an IAM Role that references the policy we created earlier.
 
 **Note:** We will use `eksctl` in this guide to create the IAM Role. The serviceaccount creation will be done by the Helm chart so that appropriate RBAC is assigned by the chart itself. If you wish to use your own serviceaccount, then make sure all the necessary rolebinding and clusterrolebinding are present.
 
 ```bash
-eksctl create iamserviceaccount --name nirmata-aws-adapter  --namespace nirmata-aws-adapter --cluster <cluster-name> --role-name nirmata-adapter-role --attach-policy-arn arn:aws:iam::111122223333:policy/kyverno-aws-adapter-policy   --role-only  --approve
+eksctl create iamserviceaccount --name nirmata-aws-adapter  --namespace nirmata-aws-adapter --cluster <cluster-name> --role-name nirmata-adapter-role --attach-policy-arn arn:aws:iam::<account-id>:policy/kyverno-aws-adapter-policy   --role-only  --approve
```
Comment (kumar-mallikarjuna) [inline docs/getting_started.md:87]: Same here.

**Review by kumar-mallikarjuna — APPROVED** [review]
LGTM, minor comments.

File: docs/getting_started.md (hunk 1 above)
Comment (kbeniwal) [reply docs/getting_started.md]: Done. Thanks!

File: docs/getting_started.md [hunk 3]
```diff
@@ -141,14 +148,16 @@ You can check the status of your configuration with:
...
-Perform the below steps to see if the adapter is installed correctly
+
+Perform the below steps to see if the adapter is installed correctly.
+
 ```bash
-# check if the controller pod is running
+# check if the controller pod is running.
 kubectl get pods -n nirmata-aws-adapter
 NAME                                              READY   STATUS    RESTARTS   AGE
 aws-adapter-kyverno-aws-adapter-997f45bb9-c2z5j   1/1     Running   0          109m
```
Comment (anusha94) [inline docs/getting_started.md:158]: Can you change the pod name to whatever is the latest convention?

Comment (kbeniwal) [reply docs/getting_started.md]: Done.

File: docs/getting_started.md [hunk 4]
```diff
@@ -141,15 +148,17 @@ You can check the status of your configuration with:
...
-aws-adapter-kyverno-aws-adapter-997f45bb9-c2z5j   1/1     Running   0          109m
+NAME                                  READY   STATUS    RESTARTS   AGE
+kyverno-aws-adapter-997f45bb9-c2z5j   1/1     Running   0          109m
 
-# check the awsadapterconfig status
+# check the awsadapterconfig status.
 kubectl get awsacfg -n nirmata-aws-adapter
-NAME                             ...   CLUSTER NAME     ...  STATUS
-aws-adapter-kyverno-aws-adapter  ...   cluster-name     ...  success
+NAME                 ...   CLUSTER NAME     ...  LAST POLLED STATUS
```
Comment (anusha94) [inline docs/getting_started.md:162]: I thought `Last Polled` was supposed to indicate the time at which it was last polled

Comment (kbeniwal) [reply docs/getting_started.md:162]: This is what we have right now,
image

Comment (anusha94) [reply docs/getting_started.md:162]: Oh I thought those were 2 different columns

**Review by anusha94 — APPROVED** [review]

**Pull request merged**